translations/fr/get5.phrases.txt

```diff
@@ -296,4 +408,64 @@
     {
         "fr"        "Le veto commencera dans {1} secondes."
     }
+    "SurrenderCommandNotEnabled"
+    {
+        "fr"        "La commande de capitulation n'est pas acctivée."
+    }
+    "SurrenderMinimumRoundDeficit"
+    {
+        "fr"        "Vous devez avoir terminé {1} manche(s) pour pouvoir capituler."
+    }
+    "SurrenderInitiated"
+    {
+        "fr"        "A vote to surrender was initiated by {1}. Your team must reach {2} votes within {3} seconds."
```

Comment:
This is not French.

Reply:
Un vote pour abandonner à été initié par {1}. Votre équipes doivent avoir {2} votes dans les {3} secondes.